src/Components/Header.js

```diff
@@ -1,13 +1,57 @@
 import styled from "styled-components";
+import { Link, useParams, useNavigate } from "react-router-dom";
 
 function Header() {
-  return <StyledHeader>Header</StyledHeader>;
+  const { noteId } = useParams();
+  const navigate = useNavigate();
+
+  return (
+    <StyledHeader>
+      {noteId && (
+        <BackButton onClick={() => navigate(-1)}>
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            fill="none"
+            viewBox="0 0 24 24"
+            strokeWidth={1.5}
+            stroke="currentColor"
+          >
+            <path
+              strokeLinecap="round"
+              strokeLinejoin="round"
+              d="M9 15L3 9m0 0l6-6M3 9h12a6 6 0 010 12h-3"
+            />
+          </svg>
```

Comment:
at some point it might be better to refactor your svgs into their own file, for now it is ok.

Reply:
I will probably throw away many of them soon, so I will move them into their own file later.